substrate/frame/support/procedural/src/pallet/expand/storage.rs

```diff
@@ -418,11 +417,14 @@ pub fn expand_storages(def: &mut Def) -> proc_macro2::TokenStream {
 		let full_ident = quote::quote_spanned!(storage.attr_span => #ident<#gen> );
 
 		let cfg_attrs = &storage.cfg_attrs;
-
+		let deprecation =
+			crate::deprecation::get_deprecation(&quote::quote! { #frame_support }, &storage.attrs)
+				.unwrap_or_else(syn::Error::into_compile_error);
 		quote::quote_spanned!(storage.attr_span =>
 			#(#cfg_attrs)*
 			{
 				<#full_ident as #frame_support::storage::StorageEntryMetadataBuilder>::build_metadata(
+					#deprecation,
```

Comment:
This will also lead to other compile errors.

Reply:
Now code does an early return if deprecation_attribute error is encountered